File: ios/Classes/GoogleMapsPlusPlugin.swift
import Flutter
import UIKit

public class GoogleMapsPlusPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let playbackFactory = GoogleMapsPlusPlaybackFactory(messenger: registrar.messenger())
    registrar.register(playbackFactory, withId: "br.com.cpndntech.google_maps_plus/playback")
    
    let plusFactory = GoogleMapsPlusFactory(registrar: registrar)
    registrar.register(plusFactory, withId: "br.com.cpndntech.google_maps_plus/plus")
  }
}

File: android/src/main/kotlin/br/com/cpndntech/google_maps_plus/GoogleMapsPlusPlugin.kt
package br.com.cpndntech.google_maps_plus

import io.flutter.embedding.engine.plugins.FlutterPlugin

class GoogleMapsPlusPlugin : FlutterPlugin {
    override fun onAttachedToEngine(flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
        flutterPluginBinding.platformViewRegistry.registerViewFactory(
            "br.com.cpndntech.google_maps_plus/playback",
            GoogleMapsPlusPlaybackFactory(flutterPluginBinding.binaryMessenger)
        )
        flutterPluginBinding.platformViewRegistry.registerViewFactory(
            "br.com.cpndntech.google_maps_plus/plus",
            GoogleMapsPlusFactory(flutterPluginBinding.binaryMessenger)
        )
    }

    override fun onDetachedFromEngine(binding: FlutterPlugin.FlutterPluginBinding) {}
}
